modules/aspect_rules_ts/metadata.json

```diff
@@ -18,6 +24,11 @@
             "email": "sahin@aspect.build",
             "github": "thesayyn",
             "github_user_id": 20563340
+        },
+        {
```

Comment:
It seems like Alex being duplicated in here caused only the first to have the `github_user_id` get appended. That might be considered a BCR bug? Should also be fixed by https://github.com/aspect-build/rules_ts/pull/823